### DISCUSSION TITLE
Combobox - multiple

### DISCUSSION THREAD

**Discussion by depsimon** [tailwindlabs/headlessui#1131] [Ideas]
First I'd like to thank you for the new Combobox component. So much better than my home-cooked search input in the listbox component.

Now, I'm wondering if you had plans to make it work with multiple (Array) values?

I am able to do it but it's not ideal as selecting an option closes the `ComboboxOptions` panel. 
I also can't rely on the `selected` slot prop of the `ComboboxOption` and need to check if it's a multiple select. Same for the `display-value` function of the `ComboboxInput`.

I believe it could be implemented based on the type of the value (Array = multiple, not array = single value).

**Comment by adigh** [comment]
I'm also interested in multiple options. I'm currently using React-Select but would love to use the Combobox (which, by the way, I think it's great!) for this.

**Comment by CricketLaChica** [comment]
You'll probably be better off creating a fork with this PR https://github.com/tailwindlabs/headlessui/pull/648 from last year (that's what we did). This has been a highly requested feature that's more than a year old already for the `Listbox` component – so I doubt it'll make it on this new Combobox without that one first.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

I'm working on multi-value support for both the Listbox and the Combobox.

**Comment by depsimon** [comment]
Seems like it's been merged by #1243

Can't wait for next release 🙌

**Comment by brielov** [comment]
Hey guys, any idea on when will this be released? I see it has already been merged and really need this feature for my current project.

**Comment by andreasvirkus** [comment]
Are there any known issues with this atm? I'm passing in an array in the `v-model` and using the `multiple` prop, but it still closes after selecting each value (multiple values are supported though)
